### PR TITLE
Max total size

### PR DESCRIPTION
**Pull request by danielcarletti**
Added lock for max total size (50mb). It will abort if the files are larger than 50mb.

### CODE REVIEW

File: bin/rise [hunk 1]
```diff
@@ -41,8 +41,12 @@ if Rise::Util.first_run?
 end
 Rise::Util.check_for_update!
 result_url = ''
-uploader = Rise::Transport::Uploader.new(options[:directory]) unless options[:directory].nil?
-uploader = Rise::Transport::Uploader.new(Dir.pwd) if options[:directory].nil?
+uploader = Rise::Transport::Uploader.new(options[:directory] || Dir.pwd)
+
+if uploader.total_files_size > 52428800
+  puts Paint["Max file size reached (#{uploader.total_files_size} > 52428800)", '#FF0000']
```
Comment (cbrnrd) [inline bin/rise]: Maybe change `52428800` to `50MB`

**Review by cbrnrd — CHANGES_REQUESTED** [review]

File: bin/rise (hunk 1 above)
Comment (danielcarletti) [reply bin/rise]: Fixed @cbrnrd

Comment (cbrnrd) [reply bin/rise]: great, thanks. I’ll test this as soon as possible and merge it if it works well

File: lib/core/transport.rb [hunk 2]
```diff
@@ -10,16 +10,17 @@ module Rise
   module Transport
     # Handles uploading files
     class Uploader
-      attr_reader :folder_path, :total_files, :include_folder
+      attr_reader :folder_path, :total_files, :include_folder, :total_files_size
       attr_reader :uuid, :current_file
```
Comment (cbrnrd) [inline lib/core/transport.rb]: Move `:total_files_size` to line 14 to keep rubocop happy

File: bin/rise (hunk 1 above)
Comment (cbrnrd) [reply bin/rise]: With your most recent changes, im getting: 
```
bin/rise:47:in `<main>': undefined method `megabytes' for 50:Integer (NoMethodError)
```

I think keep the original `if uploader.total_files_size > 52428800` but change the `52428800` in the puts statement to 50MB

File: bin/rise [hunk 3]
```diff
@@ -43,8 +43,8 @@ Rise::Util.check_for_update!
 result_url = ''
 uploader = Rise::Transport::Uploader.new(options[:directory] || Dir.pwd)
 
-if uploader.total_files_size > 50.megabytes
-  puts Paint["Max file size reached (#{uploader.total_files_size})", '#FF0000']
+if uploader.total_files_size > 52428800
+  puts Paint["Max file size reached (#{uploader.total_files_size} > 50mb)", '#FF0000']
```
Comment (cbrnrd) [inline bin/rise]: `50MB` (or MiB) not `mb` (milibits)

**Comment by cbrnrd** [comment]
Changes look good to me, merging now. Thanks for the contribution!

**Pull request merged**